tests/stdlib/thttpclient_standalone.nim

```diff
@@ -0,0 +1,32 @@
+discard """
+  cmd: "nim c --threads:on $file"
+  output: "OK"
+"""
+
+import asynchttpserver, httpclient, asyncdispatch
+
+block: # issue #16436
```

Comment:
https://nim-lang.github.io/Nim/contributing.html
> Always refer to a GitHub issue using the following exact syntax: bug #1234 as shown above, so that it's consistent and easier to search or for tooling